common/app/conf/switches/FeatureSwitches.scala

```diff
@@ -495,4 +495,14 @@ trait FeatureSwitches {
     sellByDate = new LocalDate(2018, 10, 17),
     exposeClientSide = true
   )
+
+  val Pc = Switch(
+    SwitchGroup.Feature,
+    "pc",
```

Comment:
Can we set a full name and description, so people not familiar with the project still understand what it is about? Thanks